crates/ruff_python_ast/src/comparable.rs

```diff
@@ -592,10 +592,26 @@ pub struct ExprCall<'a> {
 #[derive(Debug, PartialEq, Eq, Hash)]
 pub struct ExprFormattedValue<'a> {
     value: Box<ComparableExpr<'a>>,
+    debug_text: Option<DebugText<'a>>,
```

Comment:
Would it be possible to store a reference to a debug text instead of defining an extra `DebugText` node?
```suggestion
    debug_text: Option<&'a DebugText>,
```